docs/api/4.0-SNAPSHOT.md

```diff
@@ -0,0 +1,81 @@
+# API Docs - v4.0-SNAPSHOT
```

Comment:
Please remove this file

Reply:
Removed in commit 58653df84cce376469a0c46510ce0bc4f5036a71